roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -153,7 +153,7 @@ rbac_enabled: "{{ 'RBAC' in authorization_modes or kubeadm_enabled }}"
 
 ## List of key=value pairs that describe feature gates for
 ## the k8s cluster.
-kube_feature_gates: []
+kube_feature_gates: ['Initializers=true']
```

Comment:
This will cause deployments of kube_version < 1.8 to fail as there is not `Initializers` feature-gate on those versions.